opendbc/safety/safety.h

```diff
@@ -824,6 +824,70 @@ bool steer_angle_cmd_checks(int desired_angle, bool steer_control_enabled, const
   return violation;
 }
 
+static float get_curvature_factor(const float speed, const AngleSteeringParams params) {
+  return 1. / (1. - (params.slip_factor * (speed * speed))) / params.wheelbase;
+}
+
+bool steer_angle_cmd_checks_vm(int desired_angle, bool steer_control_enabled, const AngleSteeringLimits limits,
```

Comment:
seems this doesn't have any real time checks. we absolutely need these, e.g. what happens if I send the steer command at 100Hz?

Reply:
We never had these for angle limiting, but I thought about adding a generic frequency param for relevant tx messages to check for this, and removes torque's duplicate limits * 1.2 as well